crates/sallyport/tests/sallyport_enarxcall_tests.rs

```diff
@@ -1,6 +1,10 @@
 // SPDX-License-Identifier: Apache-2.0
 
-use super::run_test;
+#![feature(c_size_t, core_ffi_c)]
```

Comment:
This could have been specified once in a top-level module, just like before, now that's just boilerplate we have to duplicate everywhere